torchvision/transforms/v2/_misc.py

```diff
@@ -278,10 +280,27 @@ def __init__(
                 f"Got {labels_getter} of type {type(labels_getter)}."
             )
 
+    @staticmethod
+    def _get_dict_or_second_tuple_entry(inputs):
+        # Often, datasets outputs are plain dicts like {"img": ..., "labels": ..., "bbox": ...}
```

Comment:
> Often

Based on what? So far we only have the other style.

Reply:
edited comment :)